src/AutoPilotPlugins/PX4/PX4TuningComponentPlaneRate.qml

```diff
@@ -0,0 +1,164 @@
+
+        QGCLabel {
+            textFormat:         Text.RichText
+            text:               qsTr("Thrust curve <b><a href=\"https://docs.px4.io/master/en/config_mc/pid_tuning_guide_multicopter.html#thrust-curve\">?</a></b>:")
+            onLinkActivated:    Qt.openUrlExternally(link)
+            visible:            _thrustModelFactor
+        }
+        FactTextField {
+            fact:               _thrustModelFactor
+            visible:            _thrustModelFactor
+        }
+    }
+    PIDTuning {
+        width: availableWidth
+        id:    pidTuning
+
+        property var roll: QtObject {
+            property string name: qsTr("Roll")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.rollRate.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.rollRate.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Porportional gain (FW_RR_P)")
+                    description:    qsTr("Porportional gain.")
+                    param:          "FW_RR_P"
+                    min:            0.3
+                    max:            1
+                    step:           0.05
+                }
+                ListElement {
+                    title:          qsTr("Differential Gain (FW_RR_D)")
+                    description:    qsTr("Damping: increase to reduce overshoots and oscillations, but not higher than really needed.")
+                    param:          "FW_RR_D"
+                    min:            0.0004
+                    max:            0.01
+                    step:           0.0002
+                }
+                ListElement {
+                    title:          qsTr("Integral Gain (FW_RR_I)")
+                    description:    qsTr("Generally does not need much adjustment, reduce this when seeing slow oscillations.")
+                    param:          "FW_RR_I"
+                    min:            0.0
+                    max:            0.5
+                    step:           0.025
+                }
+            }
+        }
+        property var pitch: QtObject {
+            property string name: qsTr("Pitch")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.pitchRate.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.pitchRate.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Porportional Gain (FW_PR_P)")
+                    description:    qsTr("Porportional Gain.")
+                    param:          "FW_PR_P"
+                    min:            0.3
+                    max:            3
+                    step:           0.05
+                }
+                ListElement {
+                    title:          qsTr("Differential Gain (FW_PR_D)")
```

Comment:
@Jaeyoung-Lim Since when do we have this for fixed wing?

Reply:
Since https://github.com/PX4/PX4-Autopilot/pull/19912 ?